skt/executable.py

```diff
@@ -887,6 +911,16 @@ def check_args(parser, args):
                 'reporter were provided'
             )
 
+    # Check required arguments for 'run'
+    if args._name == 'run':
+
+        if not args.jobtemplate:
+            parser.error(
+                "A job template XML file is required to run jobs in Beaker. "
+                "Use the --beaker-job-template argument to specify the path "
+                "to a valid file."
+            )
```

Comment:
Mmm I understand the mismatch now, it wouldn't be better a custom action to validate this?